ui/tests/integration/components/json-editor-test.js

```diff
@@ -122,12 +123,27 @@ module('Integration | Component | json-editor', function (hooks) {
     assert.dom('.CodeMirror-code').hasText(`{ "test": "********"}`, 'shows data with obscured values');
     assert.dom('[data-test-toggle-input="revealValues"]').isNotChecked('reveal values toggle is unchecked');
     await click('[data-test-toggle-input="revealValues"]');
-    assert.dom('.CodeMirror-code').hasText(JSON_BLOB, 'shows data with real values');
+    assert.strictEqual(codemirror().getValue(), JSON_BLOB, 'shows data with real values');
```

Comment:
hasText does not properly handle whitespaces in json blobs. Used getValue instead.

Reply:
They are two different things, right? `hasText` gets what the screen sees and `getValue()` gets what the codemirror has stored in state, so I think it's important to test both cases since that's where the bug originally came from, right?